clang/test/CodeGen/coverage-profile-raw-version.c

```diff
@@ -0,0 +1,9 @@
+// RUN: %clang_cc1  -fprofile-instrument=clang -fcoverage-mapping -emit-llvm -o - %s | FileCheck %s
+// RUN: %clang_cc1 -mllvm -profile-correlate=binary -fprofile-instrument=clang -fcoverage-mapping -emit-llvm -o - %s | FileCheck %s --check-prefix=BIN-CORRELATE
```

Comment:
Specify an explicit `-target-triple` to test a COMDAT triple (e.g. Linux) and a non-COMDAT triple (e.g. macOS).

Change `// CHECK: @__llvm_profile_raw_version = {{.*}} i64 9` to test more properties including the visibility and `comdat`